deps/unwind.version

```diff
@@ -0,0 +1,7 @@
+## jll artifact
+UNWIND_JLL_NAME := LibUnwind
+UNWIND_JLL_VER  := 1.5.0+1
```

Comment:
This is redundant: https://github.com/JuliaLang/julia/pull/45729#discussion_r900798598
```suggestion
```